crf/build.gradle

```diff
@@ -27,17 +27,14 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
-    buildToolsVersion '28.0.3'
+    buildToolsVersion '29.0.3'
 }
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    
-    implementation project(':backbone'), {
-        exclude group: 'joda-time', module: 'joda-time'
-        exclude group: 'net.zetetic', module: 'android-database-sqlcipher'
-        exclude group: 'co.touchlab.squeaky'
-    }
+
+    // Dagger needs access to some RS classes
```

Comment:
Comment is confusing since the CRF module doesn't use Dagger.

Reply:
I'm confused by it too and I wrote it.  Removed.